README.md

```diff
@@ -5,27 +5,54 @@ This repository provides a template for creating **feature specifications** for
 
 When a new feature or product is to be developed, a new pull request is created, and contributors are invited to discuss its content and propose suggestions. The goal of a feature specification is to define the expected behavior on a high level and point out corner cases that need to be addressed.
 
+The person in charge of the RP (the owner) is the person assigned to the RP. This allows for the possibility of changing it over time. To change the person assigned to the PR, this should be seen with at least the lest and the new assignee.
+
+#### Start
+
+To start a new specification, it is recommended to write the first draft by filling in only the `Summary` and `Motivation` parts. 
+
+The name of the spec must follow the pattern: `PR_number-feature-name.md`. For example, if PR number 12 is about facetting, its spec will be named `0012-facetting.md` or `0012-facetted-search.md`. It will be logical that the names of the PRs will not follow each other. This is not a problem. The fact that the name has the number of PR simplifies the fact that there is no synchronization to choose this number. 
```

Comment:
There will honestly never be a need for synchronization, PR are merged one after the other, and people talk with each other when doing so. It's cleaner if the numbers simply follow each other.

Reply:
I just took the idea from [Rust](https://github.com/rust-lang/rfcs). Are you sure about the usability? What if we would like to merge 2 specs in the same time?